source/features/clean-repo-sidebar.tsx

```diff
@@ -8,7 +8,7 @@ import features from '.';
 
 async function removeReadmeLink(): Promise<void> {
 	// Hide "Readme" link made unnecessary by toggle-files-button #3580
-	(await elementReady('.muted-link[href="#readme"]'))?.parentElement!.remove();
+	(await elementReady('.muted-link[href="#readme"], .Link--muted[href="#readme"]'))?.parentElement!.remove();
```

Comment:
They all need comments

Reply:
I think what’s happening is relatively obvious. Maybe let’s just add a global expiring TODO comment for October or something.

Reply:
Fine with me.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md

Reply:
Maybe by the fetureRenamed?